core/controllers/profile.py

```diff
@@ -89,8 +89,15 @@ def get(self, username):
             edited_exp_summary_dicts = (
                 summary_services.get_displayable_exp_summary_dicts_matching_ids(
                     user_contributions.edited_exploration_ids))
+            ### function to determine whether current user is same as user on page ###
+            if (self.username == username):
```

Comment:
No need for parenthesis in python if-statements, `if self.username == username` is sufficient.